include/wil/winrt.h

```diff
@@ -294,7 +294,7 @@ namespace wil
         }
 
         //! Returns a pointer for the buffer so it can be populated
-        wchar_t* Get() const { return const_cast<wchar_t*>(m_maker.buffer()); }
+        wchar_t *Get() const { return const_cast<wchar_t *>(m_maker.buffer()); }

```

Comment:
```suggestion
        wchar_t* Get() const { return const_cast<wchar_t*>(m_maker.buffer()); }
```
Forgot I have this power

Reply:
![HemanPowerGIF](https://user-images.githubusercontent.com/40036384/211694456-14c416a9-de1b-475b-9e53-305eed23b856.gif)